tests/fixture/tmpnet/genesis.go

```diff
@@ -149,7 +150,8 @@ func NewTestGenesis(
 	// Define C-Chain genesis
 	cChainGenesis := &core.Genesis{
 		Config:     params.AvalancheLocalChainConfig,
-		Difficulty: big.NewInt(0), // Difficulty is a mandatory field
+		Difficulty: big.NewInt(0),                              // Difficulty is a mandatory field
+		Timestamp:  uint64(upgrade.InitiallyActiveTime.Unix()), // This time enables Avalanche upgrades by default
```

Comment:
this is needed for backwards compatibility as params.AvalancheLocalChainConfig now uses avalanchego InitiallyActiveTime instead of genesis to activate avalanche upgrades (specifically ap3 which also impacts the outcome of genesis)